armi/reactor/tests/test_components.py

```diff
@@ -189,14 +199,59 @@ def test_getDimension(self):
 
 class TestUnshapedComponent(TestGeneralComponents):
     componentCls = UnshapedComponent
-    componentMaterial = "Material"
+    componentMaterial = "HT9"
     componentDims = {"Tinput": 25.0, "Thot": 430.0, "area": math.pi}
 
+    def test_getComponentArea(self):
+        # a case without thermal expansion
+        self.assertEqual(self.component.getComponentArea(cold=True), math.pi)
+
+        # a case with thermal expansion
+        self.assertEqual(
+            self.component.getComponentArea(cold=False),
+            math.pi
+            * self.component.getThermalExpansionFactor(self.component.temperatureInC)
+            ** 2,
+        )
+
+        # show that area expansion is consistent with the density change in the material
+        hotDensity = self.component.density()
+        hotArea = self.component.getArea()
+        thermalExpansionFactor = self.component.getThermalExpansionFactor(
+            self.component.temperatureInC
+        )
+
+        coldComponent = self.setUp(
+            UnshapedComponent(
+                name="coldComponent",
+                material=self.componentMaterial,
+                Tinput=self.component.inputTemperatureInC,
+                Thot=self.component.inputTemperatureInC,
+                area=math.pi,
+            )
+        )
+        coldDensity = coldComponent.density()
+        coldArea = coldComponent.getArea()
+
+        self.assertNotEqual(thermalExpansionFactor, 1)
```

Comment:
Would this perhaps be more clear if it said `self.assertGreater(thermalExpansionFactor, 1)`?

Reply:
Sure, I could see that. I'll change.